### ISSUE TITLE
Analytics opt-in/out is way too biased towards opt-out

### ISSUE THREAD

**Issue by ara4n** [vector-im/element-web#15400]
The big 'No' button is red and exciting while the 'I want to help' is too wordy, and much less eye catching.  My autonomous reflex is always to hit No now, but previously i used to always hit Yes as the most 'glowing' button

<img width="303" alt="Screenshot 2020-11-04 at 14 58 27" src="https://user-images.githubusercontent.com/1294269/98127135-41157480-1eae-11eb-84c7-e35ea09898a6.png">

**Comment by niquewoodhouse** [comment]
Some quick ideas, also looked at trying to make the general copy slightly simpler and more accessible for a broad range of users.

<img width="890" alt="analytics-opt-in-copy-1" src="https://user-images.githubusercontent.com/1587166/98133788-eed85180-1eb5-11eb-98d1-69c633e890b2.png">

**Comment by ara4n** [comment]
Of these options the one which works best for me is probably this one:

<img width="285" alt="Screenshot 2020-11-04 at 16 03 15" src="https://user-images.githubusercontent.com/1294269/98135017-46c38800-1eb7-11eb-8a20-63b0fb2f9016.png">

However, i think the main thing that might be tripping me up is perhaps the contrast of the green-on-light-green button is perceptually lighter for me than the contrast on the red-on-light-red - so it almost feels greyed out.  The red button just feels much more eye catching and stronger, as if that's the option we expect people to press.

**Comment by niquewoodhouse** [comment]
I can see that.  Have we considered altering the way we present primary/secondary in binary conditions, where there's just two options in a toast?

E.g 
<img width="890" alt="analytics-opt-in-copy-2" src="https://user-images.githubusercontent.com/1587166/98146460-7a0c1400-1ec3-11eb-951a-be139da2caac.png">
